packages/lit-dev-content/site/docs/components/events.md

```diff
@@ -42,14 +42,6 @@ private _handleTouchStart(e) { console.log(e.type) }
 
 </div>
 
-If you're not using decorators, you can customize event listener options by passing an object to the event listener expression. The object must have a `handleEvent()` method and can include any the options that would normally appear in the `options` argument to `addEventListener()`.
-
-```js
```

Comment:
Should this have had both JS and TS versions anyway, even if they're the same?

Reply:
don't think so. There seems to be another example in this page just like this